Dockerfile

```diff
@@ -45,7 +45,8 @@ ENV NEXT_PUBLIC_MATOMO_SITE_ID ${NEXT_PUBLIC_MATOMO_SITE_ID}
 ENV SENTRY_DSN ${ENTRY_DSN}
 
 # Building app
-RUN npm run build
+RUN yarn install
```

Comment:
thanks MonikaCat, I suggest combine two RUN into one for a slightly smaller docker image size
i.e. `yarn install && yarn build`
https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#minimize-the-number-of-layers

Reply:
@teamchong Ok I have updated the Dockerfile, can you review again pls? Thanks ^^